.gitignore

```diff
@@ -138,4 +138,11 @@ dmypy.json
 cython_debug/
 
 # Editor
-.vscode
\ No newline at end of file
+.vscode
+
+#Courses
+**/cc451/*
+
+#results
+output.txt
+parser_output
```

Comment:
Please make a PR with this gitignore file. 

https://github.com/github/gitignore/blob/master/Python.gitignore

Reply:
I checked the diff between the .gitignore in this branch and the one in the provided link and they are identical, except for a few extra lines in this file.